packages/common/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "dist"
+  },
+  "include": ["src"]
```

Comment:
nit: some directories have `"include": ["src"]`, while others have `"include": ["./src/*.ts"]`. Should these all use the *.ts glob version instead?

Reply:
I think this is because some have subdirectories, so "src" is the more general variant